tests/test_job.py

```diff
@@ -141,16 +170,23 @@ def test_run_to_completion_cancelled(self, mock_get_status, mock_create_namespac
     @patch("kubernetes.client.BatchV1Api.create_namespaced_job")
     def test_run_to_completion_error(self, mock_create_namespaced_job, mock_check_cancelled,
                                          mock_job_delete,mock_list_namespaced_pod,  mock_read_namespaced_job):
+        """
+        Testing if the Job is in pending state when the status of the pod is pending
+        state and reason is ImagePullBackOff
+        """
         for executor in self.data['executors']:
             jobname = executor['metadata']['name']
             job = Job(executor, jobname, taskmaster.args.namespace)
             status = job.run_to_completion(1, taskmaster.check_cancelled,
-                                           120)
+                                           600)
```

Comment:
Does that mean that anybody running this test will have to wait 10 minutes? If so - that is not really acceptable, especially that you can test it in 2 seconds.

Reply:
Yes, the test case will run for 10 minutes. I will change the timestamp on the `list_namespaced_pod_error` method to avoid this.